text/0060-tasks-api.md

```diff
@@ -58,10 +58,10 @@ The motivation is to stabilize the current `update` resource to a version that c
 | field   | type    | description                     |
 |---------|---------|---------------------------------|
 | uid      | integer | Unique sequential identifier           |
-| indexUid | string | Unique index identifier |
+| indexUid | string | Unique index identifier. This field is defined to `null` for a `dumpCreation`. |
```

Comment:
```suggestion
| indexUid | string | Unique index identifier. This field is `null` when the task type is `dumpCreation`. |
```

Should we say `string or null` in the type column?

Reply:
Good point will mention that!